hack/install-kwok.sh

```diff
@@ -0,0 +1,102 @@
+set -euo pipefail
+
+# get the latest version
+KWOK_REPO=kubernetes-sigs/kwok
+# KWOK_LATEST_RELEASE=$(curl "https://api.github.com/repos/${KWOK_REPO}/releases/latest" | jq -r '.tag_name')
```

Comment:
Should this still be uncommented?

Reply:
Nope, I'll fix this.

Reply:
Actually, on trying to use the latest version, this script doesn't work. In the spirit of time, I'll keep this commented out, and just depend on the v0.3.0 version, adjusting/upgrading this script when we need to. WDYT?

Reply:
> Actually, on trying to use the latest version, this script doesn't work

Do you know why it doesn't work? I'm surprised that the installation docs that are vended by the kwok installation guide don't work correctly.